scripts/dangers/bundle-sizes.js

```diff
@@ -68,14 +68,20 @@ function getGzipSize(filePath) {
       callback();
     },
   });
+
   return new Promise((res, rej) => {
-    fs.createReadStream(filePath)
-      .pipe(zlib.createGzip({ level: 9 }))
-      // writable streams do not fire the 'close' event anymore
-      // https://github.com/nodejs/node/issues/21122#issuecomment-414622251
-      .on('close', () => res(bytesWritten))
-      .pipe(byteCounter)
-      .on('error', rej);
+    pipeline(
```

Comment:
This suggestion actually came from [James Snell](https://github.com/jasnell). He is on the Node.js TSC.

Reply:
Nice name drop lol